test/shopify_liquid_test.rb

```diff
@@ -12,7 +12,7 @@ def test_deprecated_filter_alternatives
   end
 
   def test_filter_labels
-    assert_equal(171, ThemeCheck::ShopifyLiquid::Filter.labels.size)
+    assert_equal(137, ThemeCheck::ShopifyLiquid::Filter.labels.size)
```

Comment:
The following filters in `shopify_liquid/filters.yml` are missing in `ShopifyLiquid::SourceIndex.filters.map(&:name)`:

```ruby
YAML.load(File.read("./data/shopify_liquid/filters.yml")).values.flatten - ShopifyLiquid::SourceIndex.filters.map(&:name)

[
  "h",
  "installments_pricing",
  "sentence",
  "t",
  "app_block_path_for",
  "dev_shop?",
  "app_extension_path_for",
  "global_block_type?",
  "app_block_path?",
  "app_extension_path?",
  "app_snippet_path?",
  "registration_uuid_from",
  "handle_from",
  "camelcase",
  "format_code",
  "handle",
  "encode_url_component",
  "recover_password_link",
  "delete_customer_address_link",
  "edit_customer_address_link",
  "cancel_customer_order_link",
  "unit",
  "weight",
  "paragraphize",
  "excerpt",
  "pad_spaces",
  "distance_from",
  "theme_url",
  "link_to_theme",
  "_online_store_editor_live_setting",
  "debug"
]
```

Reply:
And `Liquid::StrainerFactory.global_filter_names` still returns 168 filters :(